### PR TITLE
[BigQuery] added user agent

### PR DESCRIPTION
**Pull request by MRzNone**

### CODE REVIEW

File: jupyterlab_bigquery/jupyterlab_bigquery/details_handler/service.py [hunk 1]
```diff
@@ -161,7 +163,9 @@ class BigQueryService:
 
   _instance = None
 
-  def __init__(self, client=bigquery.Client()):
+  def __init__(self,
+               client=bigquery.Client(client_info=ClientInfo(
+                   user_agent='jupyterlab_bigquery/{}'.format(VERSION)))):
```
Comment (cbwilkes) [inline jupyterlab_bigquery/jupyterlab_bigquery/details_handler/service.py:168]: For all of these can you just set it to `'jupyterlab_gcpextension'`

**Review by cbwilkes — CHANGES_REQUESTED** [review]

File: jupyterlab_bigquery/jupyterlab_bigquery/pagedAPI_handler/pagedQuery_handler.py [hunk 2]
```diff
@@ -27,7 +29,8 @@ def __init__(self, application, request, **kwargs):
     self.pool = Pool(NUM_THREADS)
 
     if PagedQueryHandler.client is None:
-      PagedQueryHandler.client = bigquery.Client()
+      PagedQueryHandler.client = bigquery.Client(client_info=ClientInfo(
+          user_agent='jupyterlab_gcpextension/{}'.format(VERSION)))
```
Comment (cbwilkes) [inline jupyterlab_bigquery/jupyterlab_bigquery/pagedAPI_handler/pagedQuery_handler.py:33]: Also take away the `{}'.format(VERSION)`. Just use `bigquery.Client(client_info=ClientInfo(user_agent='jupyterlab_gcpextension')`

**Review by cbwilkes — CHANGES_REQUESTED** [review]

File: jupyterlab_bigquery/jupyterlab_bigquery/details_handler/service.py (hunk 1 above)
Comment (cbwilkes) [reply jupyterlab_bigquery/jupyterlab_bigquery/details_handler/service.py]: Per our discussion set `user_agent` to `'jupyterlab_gcpextension/jupyterlab_bigquery-{}'.format(VERSION)`

**Comment by MRzNone** [comment]
Discussed offline, will resolute to using `'jupyterlab_gcpextension/jupyterlab_bigquery-{}'.format(VERSION)`

**Review by cbwilkes — APPROVED** [review]

**Comment by cbwilkes** [comment]
Submitting for now but in the future we should probably create the Client info in a central location

**Pull request merged**